FileContentOrdering.md

```diff
@@ -16,23 +16,31 @@ public class House {
 
     // public functions
     public init {
-    ...
+    // ...
```

Comment:
nit: I also added comment lines here so that our sample code doesn't have any obvious syntax errors. I realize it likely wasn't confusing anyone, but figured since I was already modifying it wouldn't hurt.

Reply:
I'm in favor of that or ideally adding text into the comments to explain significance. if there isn't any, we can just omit entirely

Reply:
Omitting entirely doesn't always make sense. In this particular example if you're saying your implementation is `public init {}`, that's invalid code.